test/conftest.py

```diff
@@ -5,10 +5,7 @@
 cov = coverage(source=('doubles',))
 cov.start()
 
-try:
-    from doubles.pytest_plugin import pytest_runtest_protocol  # noqa
-except ImportError:
-    from doubles.pytest_plugin import pytest_runtest_call  # noqa
+from doubles.pytest_plugin import pytest_pyfunc_call  # noqa
```

Comment:
for reusing plugin modules explicitly the recommended pattern is to set `pytest_plugins = ['doubles.pytest_plugin']` instead of importing parts (this ensures the whole plugin gets used without having to track what to import)